charactersheet/charactersheet/templates/character/magic_items.tmpl.html

```diff
@@ -56,9 +56,8 @@
               data-target="#viewMagicItem">
           </td>
           <td data-bind="text: magicItemDescription, click: $parent.editItem"
-            href="#" class="hidden-sm hidden-xs"
-            data-toggle="modal"
-            data-target="#viewMagicItem">
+            href="#" class="hidden-sm hidden-xs table-cell-overflow-hide"
+            data-toggle="modal" data-target="#viewMagicItem">
          </td>
```

Comment:
I noticed you are hiding the column from sm and xs devices.  However perhaps the column header should be hidden too?